OpenSim/Simulation/SimbodyEngine/Joint.cpp

```diff
@@ -417,21 +429,23 @@ SimTK::SpatialVec Joint::calcEquivalentSpatialForce(const SimTK::State &s,
     return FB_G;
 }
 
-/** Joints only produce power when internal constraint forces have components along
-    the mobilities of the joint (for example to satisfy prescribed motion). In 
-    which case the joint power is the constraint forces projected onto the mobilities
-    multiplied by the mobilities (internal coordinate velocities). Only constraints
-    internal to the joint are accounted for, not external constraints that effect
-    joint motion. */
+/** Joints only produce power when internal constraint forces have components 
+along the mobilities of the joint (for example to satisfy prescribed motion). 
+In which case the joint power is the constraint forces projected onto the 
+mobilities multiplied by the mobilities (internal coordinate velocities). Only 
+constraints internal to the joint are accounted for, not external constraints 
+that effect joint motion. */
```

Comment:
A constraint on its own cannot effect joint motion. I think "affect" is intended.